pxc-80/dockerdir/etc/mysql/node.cnf

```diff
@@ -2,14 +2,9 @@
 
 datadir=/var/lib/mysql
 socket=/tmp/mysql.sock
-log-error=/var/log/mysqld.log
 pid-file=/var/run/mysqld/mysqld.pid
-expire_logs_days=7
-log_error_verbosity=3
 skip-host-cache
 
-symbolic-links=0
-
 #server_id=0
 binlog_format=ROW
 default_storage_engine=InnoDB
```

Comment:
Could you please also remove wsrep_log_conflicts wsrep_debug

Reply:
done